app/views/facility_user_reservations/index.html.haml

```diff
@@ -0,0 +1,66 @@
+= content_for :h1 do
+  = current_facility
+
+= content_for :sidebar do
+  = render "admin/shared/sidenav_users", sidenav_tab: "users"
+
+= content_for :tabnav do
+  = render "admin/shared/tabnav_users", secondary_tab: "reservations"
+
+%h1= text("users.reservations.head", name: @user.full_name)
+
+- if @order_details.blank?
+  %p.notice= text("users.reservations.none", name: @user.full_name)
+
+- else
+  %table.table.table-striped.table-hover.old-table
+    %thead
+      %tr
+        %th.centered= OrderDetail.human_attribute_name(:id)
+        %th.centered= text("users.reservations.actions")
+        %th= text("users.reservations.timespan")
+        %th= OrderDetail.human_attribute_name(:product)
+        %th.centered= OrderDetail.human_attribute_name(:status)
+        %th.currency= OrderDetail.human_attribute_name(:actual_total)
+
+    %tbody
+      - OrderDetailPresenter.wrap(@order_details).each do |order_detail|
+        - reservation_presenter = ReservationUserActionPresenter.new(controller, order_detail.reservation)
+
+        %tr
+          %td.centered= link_to order_detail, order_detail.show_order_path
+
+          %td.centered
+            - if order_detail.reservation.try(:can_cancel?)
+              - fee = CancellationFeeCalculator.new(order_detail.reservation).fee
+              - confirm_text = fee > 0 ? text("reservations.delete.confirm_with_fee", fee: number_to_currency(fee)) : text("reservations.delete.confirm")
+
+              = link_to text("reservations.delete.link"),
+                facility_user_reservation_cancel_path(current_facility, @user, order_detail),
+                method: :put,
```

Comment:
I see the `reservation` as getting updated when canceled. More of a state change than a removal.